scripts/install.js

```diff
@@ -0,0 +1,12 @@
+var sauceConnectLauncher = require('sauce-connect-launcher');
+var SAUCE_CONNECT_LAUNCHER_VERSION = '4.5.4';
+
+sauceConnectLauncher.download({
+  logger: console.log.bind(console),
+  version: SAUCE_CONNECT_LAUNCHER_VERSION,
+}, function (err) {
+  if (err) {
+    console.log(`Failed to download sauce connect binary: ${err}`);
+    console.log('sauce-connect-launcher will attempt to re-download next time it is run.');
+  }
```

Comment:
@xwa130saucer let's raise the error so this dotnset silently fail and if possible let's change the console.log to console.error

Reply:
Added `process.exit(1)` and changed the first `.log` to `.error` since the second one is more likely written for users.